### PR TITLE
Feature/skillrank

### PR DESCRIPTION
**Pull request by odd13**
Requesting pull.

### CODE REVIEW

File: app/controllers/skills_controller.rb [hunk 1]
```diff
@@ -42,6 +42,6 @@ def update
  private
```
Comment (garethstokes) [inline app/controllers/skills_controller.rb]: use two spaces instead of tabs

File: db/migrate/20170129022439_create_skills.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ def change
 
       # TODO: consider a rank column, such that you can rate how competent
```
Comment (garethstokes) [inline db/migrate/20170129022439_create_skills.rb]: can get rid of the TODO comment

File: db/seeds.rb [hunk 3]
```diff
@@ -2,30 +2,40 @@
...
+  { name: 'VPS deployments', rank: 2 },
+  { name: 'Meteor', rank: 4 }
 ]
 
-seedArray.each do |a|
+seedArraySkill.each do |a|
   Skill.find_or_create_by(a)
 end
+
+seedArrayDisciplines = [
```
Comment (garethstokes) [inline db/seeds.rb]: %s/seedArrayDisiciplines/disiciplines/g

**Comment by garethstokes** [comment]
on the model for the Skill add code that does roughly this


```
def rgb_for_current_rank
  values = [ 
    "rgb(191,240,115)",
    "rgb(13,201,247)",
    "rgb(127,127,127)",
    "rgb(240,91,71)",
    "rgb(237,28,36)"
  ]

  values[rank]
end
```

in your view, on the instance of the skill then you need to call

```
skill.rgb_for_current_rank
```

File: app/helpers/welcome_helper.rb [hunk 4]
```diff
@@ -1,2 +1,12 @@
 module WelcomeHelper
+  #getRank returns a colour in relation to the skill rank that has ben set
+  def getRank(sk)
```
Comment (garethstokes) [inline app/helpers/welcome_helper.rb]: try to stay away from naming functions like "getThing" as it's not considered the 'railsy' way
instead just call it "thing"

File: app/controllers/skills_controller.rb (hunk 1 above)
Comment (odd13) [reply app/controllers/skills_controller.rb]: Done

File: db/migrate/20170129022439_create_skills.rb (hunk 2 above)
Comment (odd13) [reply db/migrate/20170129022439_create_skills.rb]: Done

File: app/helpers/welcome_helper.rb (hunk 4 above)
Comment (odd13) [reply app/helpers/welcome_helper.rb]: Done.
Makes sense.

File: db/seeds.rb (hunk 3 above)
Comment (odd13) [reply db/seeds.rb]: Changed to better naming conventions. Bad habbits will go!

**Pull request merged**